.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "0 2 * * *"
+defaults:
+  run:
+    # Run Git Bash on Windows. Otherwise it uses PowerShell Core and we'd need
+    # to install more dependencies. Ubuntu default shell is already Bash.
+    # @see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    env:
+      ACLI_PRINT_COMMAND_OUTPUT: 1
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+        php: [ "7.3", "7.4", "8.0"]
+        coverage: [ none ]
+        include:
```

Comment:
Not urgent, but this seems like it's running one more job than necessary. Do we need a dedicated job for this, or can me run it as part of one of the other builds?

Reply:
Took some digging, but I was able to eliminate this job without embiggening the build file, taking inspiration from https://github.com/actions/runner/issues/409